packages/ui/constants/index.ts

```diff
@@ -92,3 +92,9 @@ export const COLLATERAL_FACTOR_TOOLTIP =
   'Collateral factor can range from 0-90%, and represents the proportionate increase in liquidity (borrow limit) that an account receives by depositing the asset.';
 export const ADMIN_FEE_TOOLTIP =
   "The fraction of interest generated on a given asset that is routed to the asset's admin address as a fee.";
+export const ENABLED_CHAINS: SupportedChains[] = [
```

Comment:
This value should change for `production` & `development`.

Can we maybe somehow reuse the netlify environment variables?

```
EVMOS = "true"
BSC = "true"
MOONBEAM = "true"
POLYGON = "true"
ARBITRUM = "true"
```

Reply:
https://github.com/Midas-Protocol/monorepo/blob/ef1f83ceeb81022f233ccac6be2e27ec27b55870/netlify.toml#L20-L24

Reply:
makes sense, I will read it from `.env`